### ISSUE TITLE
Setup fails on macos

### ISSUE THREAD

**Issue by patjgardner** [rcaloras/bashhub-client#78]
I already have a bashhub account and I am setup and working successfully on Linux. However, setup on macos 10.15.2 with bash shell fails:

>./setup: line 214: 23062 Illegal instruction: 4  ../env/bin/bashhub setup

> 

Not being familiar with python I'm unable to debug the cause of the error myself.

**Comment by rcaloras** [comment]
Hey @patjgardner, thanks for opening sorry you're having an issue with install on Macos. I haven't tried to install on macos 10.15 (Catalina) before. Maybe something with the Python installation perhaps. Does this occur when trying to rerun it? You're getting pretty far with the install, I believe to: https://github.com/rcaloras/bashhub-client/blob/16dc6cf45c9300dbd4263c6c2fbcda86a99db9d1/install-bashhub.sh#L254 

Can you try rerunning that command manually and inspecting the output of the environment setup by running:

```bash
ls ~/.bashhub/env/bin
~/.bashhub/env/bin/python --version
~/.bashhub/env/bin/bashhub setup
```

**Comment by patjgardner** [comment]
>ls ~/.bashhub/env/bin
activate         activate.fish    bashhub          chardetect       easy_install-2.7 pip-2.7          python2          virtualenv
activate.csh     activate_this.py bh               easy_install     pip              python           python2.7        virtualenv-2.7

>~/.bashhub/env/bin/python --version
Python 2.7.17

> ~/.bashhub/env/bin/bashhub setup
Welcome to bashhub setup!
Are you a new user? [Y/n] n
Please enter your bashhub credentials
Username: pat
Password:
Illegal instruction: 4

**Comment by patjgardner** [comment]
Googling this issue returns many hits, and it looks like the illegal instruction is caused by a compiler optimisation not supported by my cpu (I'm running Catalina on an older unsupported Mac).

I have the same version of python 2.7.17 in my /usr/local/bin which was built using brew for my specific platform. Is it possible to point your installer to that python env instead?

**Comment by rcaloras** [comment]
Believe this is fixed in 2.2.0. @patjgardner please give it a try and report back!
